### PR TITLE
Dashboard displays orders#95

### LINKED ISSUE #95: User Story 39, Merchant Dashboard displays Orders
```
As a merchant
When I visit my dashboard ("/dashboard")
If any users have pending orders containing items I sell
Then I see a list of these orders.
Each order listed includes the following information:
- the ID of the order, which is a link to the order show page ("/dashboard/orders/15")
- the date the order was made
- the total quantity of my items in the order
- the total value of my items for that order
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #97

depends on #96

### PR DESCRIPTION
**Pull request by le3ah**
Adds new methods for detecting pending orders of each merchant.  All tests passing. Closes #95

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -33,17 +33,17 @@ def quantity_of_order
...
   end
 
-  def value_of_my_items
+  def value_of_my_items(user_id)
     OrderItem.joins(:item)
-    .where("items.user_id=#{self.id}")
-    .pluck("sum(quantity * order_items.price)")
+    .where("order_id=? AND items.user_id=?", self.id, user_id)
+    .pluck("sum(quantity * order_items.price)")[0]
   end
```
Comment (stoic-plus) [inline app/models/order.rb]: I really appreciate this! Adding the `order_id` check to the where was needed
